soymsg/pomsg/pomsg_test.go

```diff
@@ -70,7 +78,7 @@ func TestPlural(t *testing.T) {
 		return
 	}
 
-	const locale = "zz"
+	const locale = "en"
```

Comment:
Is this change still required?

Reply:
Nope, should be ok to swap back. Fixing.